src/components/Modal.css

```diff
@@ -0,0 +1,44 @@
+.modal {
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+}
+
+.modal-content {
+  background-color: rgba(0, 0, 0, 0.8);
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 40%;
+  box-shadow: 0 5px 8px 0 rgba(0, 0, 0, 0.2);
+  padding: 20px;
+  height: 400px;
+  overflow: scroll;
+  color: #fff;
+  font-family: arial;
+  font-size: 14px;
+  position: fixed;
+}
+
+.show {
```

Comment:
Recall how we "stacked" classes before -- I'd like to see that again here (ie, `.HelpModal.show`, `.HelpModal.hide`

Reply:
Done